routes/products.js

```diff
@@ -0,0 +1,100 @@
+const express = require('express');
+const router = express.Router();
+const productDB = require('../db/products.js');
+const database = productDB.getProducts().products;
```

Comment:
You shouldn't try to use the array inside productDb like this. You should stick with the methods expose through the module.exports.